src/screens/SettingsScreen.js

```diff
@@ -106,12 +118,59 @@ class SettingsScreen extends React.Component<Props> {
           >
             <Text style={styles.saveText}>Save</Text>
           </TouchableOpacity>
+          <TouchableOpacity
+            onPress={() =>
+              Alert.alert(
+                'Are you sure you want to clear your data?',
+                'This cannot be undone.',
+                [
+                  {
+                    text: 'No',
+                    onPress: () => console.log('Data Clear Canceled!'),
```

Comment:
also try to remove these print statements if you don't need them anymore. trying to maintain a cleaner codebase this time!